src/routes/safe/components/Balances/SendModal/screens/ModalHeader/index.tsx

```diff
@@ -15,15 +18,17 @@ interface HeaderProps {
   title: string
 }
 
-export const Header = ({ onClose, subTitle, title }: HeaderProps): ReactElement => {
+export const ModalHeader = ({ onClose, subTitle, title }: HeaderProps): ReactElement => {
   const classes = useStyles()
+  const connectedNetwork = useSelector(networkSelector)
```

Comment:
It should actually be `useSelector(currentChainId)`.
We should indicate the contract/chain the app is pointing to, not the wallet connection.